src/Tribe/Amalgamator.php

```diff
@@ -269,11 +269,11 @@ public static function migration_button( $text = '' ) {
 				), admin_url( 'edit.php' )
 			)
 		);
-		$url  = esc_url( add_query_arg( array( 'amalgamate' => '1' ), $url ) );
+		
```

Comment:
Super minor: codesniffer is flagging up the unnecessary whitespace here (two tabs).